integration/sawtooth_integration/docker/test_track_and_trade.yaml

```diff
@@ -26,6 +26,29 @@ services:
     command: track-and-trade-tp -vv tcp://validator:4004
     stop_signal: SIGKILL
 
+  rethinkdb:
+    image: rethinkdb
+    expose:
+      - 8080
+      - 28015
+
+  tt-server:
```

Comment:
I'm not interested in going back and refactoring right now, but going forward we should be consistent about whether we are using `tt` or `tnt` as a prefix for Track and Trade resources. I've been doing one, and you've been doing the other. Do you have an opinion @peterschwarz?

Reply:
Personally, I'm inclined to lean towards `tnt`